cli/azd/cmd/pipeline_test.go

```diff
@@ -25,7 +25,7 @@ func TestPipelineCmd(t *testing.T) {
 func TestPipelineConfigCmd(t *testing.T) {
 	command := newPipelineConfigCmd()
 	assert.EqualValues(t, "config", command.Use)
-	assert.EqualValues(t, "Create and configure your deployment pipeline by using GitHub Actions or Azure Pipelines.",
+	assert.EqualValues(t, "Create and configure your deployment pipeline by using GitHub or Azdo Actions.",
```

Comment:
This prose feels worse.  I think the product is called [Azure Pipelines](https://learn.microsoft.com/en-us/azure/devops/pipelines/?view=azure-devops).  I think what we had before was better?